### PR TITLE
Examples: Python ShellBang & Executable

### PR DESCRIPTION
**Pull request by ax3l**
Make sure that all PICMI scripts and all analysis Python scripts in `Examples/` are:
- executable (`chmod a+x`)
- start with a shell-bang to `python3`

Now, all scripts can be run directly without a `python3 ...` prefix and also default to the only right executable of Python on older systems. (New systems always have a `python3` alias, too.)

### CODE REVIEW

File: Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
+#!/usr/bin/env python3
+
 from pywarpx import picmi
-import numpy as np
```
Comment (dpgrote) [inline Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py:2]: Can you undo this change? This will cause a conflict with PR #2637. That PR reorders the imports to the standard style, with the system packages first.

**Comment by dpgrote** [comment]
Thanks for this clean up. I agree that is it better to explicitly reference `python3`. There are still too many places where `python` still means `python2`. I have just the one request.

**Review by roelof-groenewald — APPROVED** [review]
Cool. We'll keep in mind to add the ShellBang to future python scripts and make them executable.

File: Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py (hunk 1 above)
Comment (ax3l) [reply Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py:2]: Sure, no prob! :) Fixed!

**Comment by lgiacome** [comment]
Great, thanks @ax3l!

**Review by lgiacome — APPROVED** [review]

**Review by prkkumar — APPROVED** [review]

**Review by dpgrote — APPROVED** [review]
Thanks!
